main.py

```diff
@@ -95,6 +95,19 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/queue/", response_model=Dict)
+async def get_experiment_queue() -> Dict:
+    """Gets the list of queued experiment returns it.
+
+    Returns:
+        A dictionary of queued experiments with rid as their keys.
```

Comment:
Please write it down in more detail.
For example, what is each value and if the value is a dictionary, then what are its keys and values.